packages/cli-e2e/__tests__/orgResources.specs.ts

```diff
@@ -210,8 +211,10 @@ describe('org:resources', () => {
     const getResourceFolderContent = (projectPath: string) =>
       readdirSync(join(projectPath, 'resources'));
 
-    beforeEach(() => {
-      rmSync(destinationPath, {recursive: true, force: true});
+    beforeEach(async () => {
+      await retry(() => {
+        rmSync(destinationPath, {recursive: true, force: true});
```

Comment:
Why retry here

Reply:
On Windows, the directory is still flagged as "EBUSY" after the CLI execution.
This could be due to either the slowness of NTFS compared to extFS, or more likely that we don't kill the CLI properly on Windows (`fkill` works nice, but well windows can still be a pain sometimes).

So, because this is a 'setup' phase, I feel comfortable putting a retry if it increases the reliability. I would not have done so if it was part of the 'execute/verify' phases of our tests tho.